src/classes/RD2_ApiService_TEST.cls

```diff
@@ -268,6 +379,30 @@ private class RD2_ApiService_TEST {
         return (Map<Id, List<Schedule>>) JSON.deserialize(response, Map<Id, List<Schedule>>.class);
     }
 
+    /**
+     * @description Utility method to retrieve installments via Callable_API
+     * @param startDate date
+     * @param endDate date
+     * @param rdId Recurring Donation Id
+     * @return Deserialized api response
+     */
+    private static Map<Id, List<Installment>> retrieveInstallmentsUsingApi(
```

Comment:
Let's name this helper method as what the `Callable_API` name is: `queryInstallments()`. The arguments can be specified on the same line then.